packages/vats/src/core/basic-behaviors.js

```diff
@@ -422,7 +420,7 @@ export const addBankAssets = async ({
   produceIssuer.IST.resolve(runKit.issuer);
   produceBrand.BLD.resolve(bldKit.brand);
   produceBrand.IST.resolve(runKit.brand);
-  return Promise.all([
+  await Promise.all([
```

Comment:
why not return?

Reply:
In an attempt to get more static typing for bootstrap stuff, I defined a type for these "behaviors" and `Promise<void>` seemed like the right return type.